doc/source/appendix/samplecode_intro.rst

```diff
@@ -0,0 +1,41 @@
+.. _samplecode:
+
+*******************
+Code samples (stub)
```

Comment:
I'm suspicious of this file since it seems pretty generic and has 'stub' in the title. Does any of this pertain to the GA4GH APIs?